api/src/main/java/io/druid/data/input/impl/prefetch/CacheManager.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.data.input.impl.prefetch;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.druid.java.util.common.ISE;
+import io.druid.java.util.common.logger.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A class managing cached files used by {@link PrefetchableTextFilesFirehoseFactory}.
+ */
+class CacheManager<ObjectType>
```

Comment:
Why not following Java convention and not just calling parameter `T`? Same in other places

Reply:
Changed.